flake.nix

```diff
@@ -22,7 +22,7 @@
 
       rzk = "rzk";
       rzk-js = "rzk-js";
-      ghcVersion = "ghc962";
+      ghcVersion = "ghc963";
```

Comment:
Probably 964 will also be OK.

Reply:
Can we maybe use 9.6.5? It has been released yesterday :)

Reply:
I don't think it's in nixpkgs just yet

Reply:
It's in nixpkgs, but I suggest we wait some time (https://github.com/rzk-lang/rzk/issues/182), then update again. The nixpkgs version in this PR is OK for now.